### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by svmnotn**
Small pr but I caught the error so I thought I might as well

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -422,7 +422,7 @@ impl<'a> DynamicReload<'a> {
     // to read from it directly so this code does some testing first to ensure we
     // can actually read from it (by using metadata which does a stat on the file)
     // If we can't read from it we wait for 100 ms before we try again, if we can't
-    // do it with in 1 sec we give up
+    // do it within 1 sec we give up
```
Comment (svmnotn) [inline src/lib.rs]: I did not notice that Atom did all those line ending changes... well, If you want to close the pr and just push this fix its fine.

File: src/lib.rs [hunk 2]
```diff
@@ -418,11 +416,11 @@ impl<'a> DynamicReload<'a> {
...
 
-    // In some cases when a file has been set that it's reloaded it's actually not possible
+    // In some cases when a file has been set so that it's reloaded, it's actually not possible
     // to read from it directly so this code does some testing first to ensure we
-    // can actually read from it (by using metadata which does a stat on the file)
-    // If we can't read from it we wait for 100 ms before we try again, if we can't
-    // do it with in 1 sec we give up
+    // can actually read from it (by using metadata which does a stat on the file).
+    // If we can't read from it, we wait for 100 ms before we try again, if we can't
+    // do it within 1 sec we give up
```
Comment (svmnotn) [inline src/lib.rs]: I did not notice that Atom did all those line ending changes... well, If you want to close the pr and just push this fix its fine.

**Comment by emoon** [comment]
Looks good. I should really fix so I don't have any extra space on line endings. Thanks!

**Comment by svmnotn** [comment]
No problem

**Pull request merged**